api/src/pcapi/scheduled_tasks/clock.py

```diff
@@ -147,8 +149,8 @@ def pc_notify_newly_eligible_users() -> None:
     if not settings.IS_PROD and not settings.IS_TESTING:
         return
     yesterday = datetime.date.today() - datetime.timedelta(days=1)
-    for user in get_newly_eligible_users(yesterday):
-        send_newly_eligible_user_email(user)
+    for user in get_newly_eligible_age_18_users(yesterday):
+        send_newly_eligible_age_18_user_email(user)
```

Comment:
Comme pour l'autre PR, peut-être entourer chaque itération d'un try/except/log, pour ne pas compromettre l'envoi des mails suivants si ça échoue pour l'un